gradle/run.gradle

```diff
@@ -37,8 +37,31 @@ testClusters {
   }
 }
 
+testClusters {
```

Comment:
Do we really need multiple testClusters to be defined? 
We should be able to configure something like:

```
def numNodes = findProperty('numNodes') as Integer ?: 1

testClusters {
  runTask {
    testDistribution = 'archive'
    if (numNodes > 1) numberOfNodes = numNodes
  }
}
```

Reply:
Currently i have not updated this.. Though i have updated the parameter and made the check concise.
I had kept the separate testClusters configuration to keep the existing one independent of multi-node behaviour.
This can be helpful in case where we need single node cluster specific property to be defined. Let me know your thoughts.

Reply:
Yup which is why the `if` condition helps.
Set `numberOfNodes` if `numNodes` is > 1.
```
if (numNodes > 1) numberOfNodes = numNodes
```

Reply:
Updated. It works as expected.